packages/browser/src/sdk.ts

```diff
@@ -97,9 +105,17 @@ export function init(options: BrowserOptions = {}): void {
   if (options.stackParser === undefined) {
     options.stackParser = defaultStackParsers;
   }
-
   const { transport, newTransport } = setupBrowserTransport(options);
-  initAndBind(BrowserClient, options, transport, newTransport);
+
+  const clientOptions: BrowserClientOptions = {
+    ...options,
+    stackParser: stackParserFromOptions(options),
+    integrations: getIntegrationsToSetup(options),
+    // TODO(v7): get rid of transport being passed down below
+    transport: options.transport || (supportsFetch() ? FetchTransport : XHRTransport),
```

Comment:
this is a huge hack, but given we are switching to the new transports, left it as is.

Reply:
😎 https://github.com/getsentry/sentry-javascript/pull/4943/files#diff-40448f2ae3835a5923e7744870a97909f2dc739e4d2ee5a3ea0afb3459c5ec28R112